inventory/inventory.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+
+"""
+Environment variable for this inventory script:
+OL_TYPE:        required, the openlab deployment type, one of: allinone, allinone-ha, openlab, openlab-ha
+OL_{host}_IP:   new ip address specified for the {host}, such as: OL_ZUUL01_IP, OL_NODEPOOL01_IP,
+                the {host} must be one of the host name defined in the yaml inventory files.
+OL_SWITCH_MASTER_SLAVE: whether switch the master/slave IP address
+
+"""
+import os
+import sys
+
+import argparse
+import json
+import subprocess
+
+
+def parse_inventory():
+    inventory_file = 'inventory/%s.yaml' % os.environ.get('OL_TYPE')
+    parsed_inventories = subprocess.Popen(
+        ['ansible-inventory', '-i', inventory_file, '--list'],
```

Comment:
I tried this cmd in openlab zuul node, the following error raised:
root@donotdel-openlab-zuul:/home/ubuntu/labkeeper# ansible-inventory -i inventory/products.yaml --list
ERROR! Unexpected Exception, this is probably a bug: Cannot json serialize huawei123!@#

Reply:
I think the labkeeper code on the zuul node is very old, we have made many changes recently, you can try same command in allinone node.

Reply:
Please make sure this command can be executed successfully in openlab zuul node, if it's ok then ignore this comment：）